tests/polkadot_tests/calls.rs

```diff
@@ -35,11 +35,17 @@ fn revert() {
 
     runtime.function_expect_failure("test", Vec::new());
 
-    assert_eq!(runtime.output().len(), 0);
+    assert_eq!(runtime.output().len(), 4 + "yo!".to_string().encode().len());
 
     runtime.function_expect_failure("a", Vec::new());
 
-    assert_eq!(runtime.output().len(), 0);
+    assert_eq!(
+        runtime.output().len(),
+        4 + "revert value has to be passed down the stack"
+            .to_string()
+            .encode()
+            .len()
+    );
 
```

Comment:
Why not assert the contents of `runtime.output` rather than the just the length?

Reply:
Good idea, added it. However I noticed that the endianness of this function selector is swapped, something to look into afterwards.

Reply:
Another bug.

Reply:
Yes. I guess we never found out until now because it still works in the try catch when the endianess issue is consistent enough.

Reply:
Fixed in #1421